esmvaltool/diag_scripts/shared/_validation.py

```diff
@@ -0,0 +1,86 @@
+"""Load functions needed by diags with CONTROL and EXPERIMENT"""
+import os
+import logging
+import iris
+from esmvaltool.preprocessor._time_area import time_average
```

Comment:
Use the public function instead of importing from a private module: replace by 'from esmvaltool.preprocessor import time_average`